File: Part4/Python/Function_IgnoredReturn.py
def main():
    user_input = ""
    entered_integer = None
    
    print( "Type 1 and press enter." )
    user_input = input()
    
    # Function called but return value ignored
    int( user_input )
    
    print( f"The user entered the integer { entered_integer }" )
    
if __name__ == "__main__":
    main()

File: Part4/C#/Function_IgnoredReturn.cs
using System;

class Program
{
    static void Main()
    {
        string userInput;
        int enteredInteger;
        
        Console.WriteLine("Type 1 and press enter.");
        userInput = Console.ReadLine();
        
        // Function called but return value ignored
        int.Parse(userInput);
        
        // enteredInteger was never assigned a value
        Console.WriteLine("The user entered the integer {0}", enteredInteger);
    }
}
